bot/subscription/__main__.py

```diff
@@ -64,26 +94,20 @@ async def start(update: Update, context: ContextTypes.DEFAULT_TYPE) -> int:
 
 
 async def success_subscribe(update: Update, context: ContextTypes.DEFAULT_TYPE) -> int:
-    user_id = context.user_data["_id"]
-    prepare_data(context.user_data)
-    parameters = SubscriberParameters(**context.user_data)
-    subscriber = Subscriber(
-        _id=user_id,
-        parameters=parameters,
-        active=True,
-    )
-    db.subscribers.find_one_and_replace({"_id": user_id}, subscriber.dict(by_alias=True), upsert=True)
+    update_subscriber(True, context.user_data)
```

Comment:
когда аргументов мало, то лучше их именованными передавать, а то фиг знаешь что за True/False 
```suggestion
    update_subscriber(active=True, data=context.user_data)
```

Reply:
А мало это сколько?)